app/src/main/java/com/github/sdpcoachme/profile/ProfileActivity.kt

```diff
@@ -356,6 +335,7 @@ class ProfileActivity : ComponentActivity() {
                 }
             )
 
+            // TODO: only allow to send message to coaches ?
```

Comment:
As discussed together, `isViewingCoach` needs to be refactored to some `isViewingProfile`. Also now that our user profile is shown in an event participants list, if I'm a coach and click on my name, I can message myself on the `ProfileActivity` launched, which crashes the app. I've created an issue for that  #151.